DEVELOPING_OPAL/demos/src/main/scala/org/opalj/fpcf/analyses/FieldAccessInformationAnalysisDemo.scala

```diff
@@ -0,0 +1,112 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package fpcf
+package analyses
+
+import org.opalj.ai.domain.l2.DefaultPerformInvocationsDomainWithCFGAndDefUse
+import org.opalj.ai.fpcf.properties.AIDomainFactoryKey
+import org.opalj.br.Field
+import org.opalj.br.Method
+import org.opalj.br.analyses.BasicReport
+import org.opalj.br.analyses.Project
+import org.opalj.br.analyses.ProjectAnalysisApplication
+import org.opalj.br.fpcf.FPCFAnalysesManagerKey
+import org.opalj.br.fpcf.analyses.EagerFieldAccessInformationAnalysis
+import org.opalj.br.fpcf.properties.fieldaccess.FieldReadAccessInformation
+import org.opalj.br.fpcf.properties.fieldaccess.FieldWriteAccessInformation
+import org.opalj.br.fpcf.properties.fieldaccess.MethodFieldReadAccessInformation
+import org.opalj.br.fpcf.properties.fieldaccess.NoFieldReadAccessInformation
+import org.opalj.br.fpcf.properties.fieldaccess.NoFieldWriteAccessInformation
+import org.opalj.util.PerformanceEvaluation.time
+import org.opalj.util.Seconds
+
+import java.net.URL
+
+/**
+ * Runs analyses for field accesses throughout a project and automatically excludes any JDK files included in the project
+ * files from the summary at the end.
+ *
+ * @author Maximilian Rüsch
+ */
+object FieldAccessInformationAnalysisDemo extends ProjectAnalysisApplication {
+
+    private val JDKPackages = List("java/", "javax", "javafx", "jdk", "sun", "oracle", "com/sun",
+      "netscape", "org/ietf/jgss", "org/jcp/xml/dsig/internal", "org/omg", "org/w3c/dom",
+      "org/xml/sax")
+
+    override def title: String = "Determines read and write accesses to fields"
+
+    override def description: String = "Identifies fields which are never written nor read"
```

Comment:
Sounds contradictory to the `title`. Does it determine reads and writes, or unread/unwritten fields, or both?